controllers/review.controller.js

```diff
@@ -16,12 +16,15 @@ class ReviewController {
     }
 
     // Getting all reviews for a partciular movie
-    async getAllByMovieId(movieId) {
-        const reviews = await this.reviewRepository.find({ movieId:movieId }, null, {
-            sort: {
-                createdAt: -1
-            }
-        });
+    async getAllByMovieId(movieId, options) {
```

Comment:
i tried looking up on options and queryoptions on the web but couldnt find much. 
What is option as a parameter for? queryoptions ={} as an object? 
How is object data transferred to this queryOptions object?